lib/mzgl/midi/MidiCCMessageParser.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "MidiCCModeDetector.h"
+
+class MidiCCMessageParser {
```

Comment:
The `MidiCCMessageParser` is the bit that will tell you if it's a positive or negative movement on the pot. It could be extended to include "velocity", but I cant find many manufacturers who support that beyond Arturia.

Reply:
Yes we can do this in the future. I think my akai mpd218 does have acceleration.